psdk_wrapper/include/psdk_wrapper/psdk_wrapper.hpp

```diff
@@ -2127,6 +2166,8 @@ class PSDKWrapper : public rclcpp_lifecycle::LifecycleNode
   T_DjiAircraftInfoBaseInfo aircraft_base_info_;
   E_DjiCameraType attached_camera_type_;
   E_DjiLiveViewCameraSource selected_camera_source_;
+  T_DjiCameraManagerFileList media_file_list_;
```

Comment:
This is to keep track of the file list dji provides. Thats how they do it so I just copy their implementation.

Reply:
I see. But in our case we do not really need that because all what we do is to expose a srv which at the output will give you the list of files. Then is up to each user implementation to see how they want to store or use that information.